### PR TITLE
Add budget graph display

### PR DESCRIPTION
**Pull request by inbarbarkai**
* Add support for displaying budget statistics.

### CODE REVIEW

File: api/budget/user_budget/requirements.txt
Comment (yarons) [inline api/budget/user_budget/requirements.txt:1]: It's OK for now but we will specify the exact version in the future.

**Comment by yarons** [comment]
Besides my comments I think it's good to merge.

File: api/budget/user_budget/requirements.txt
Comment (inbarbarkai) [reply api/budget/user_budget/requirements.txt:1]: I will freeze once I'm done

Comment (inbarbarkai) [reply api/budget/user_budget/requirements.txt:1]: Freezed versions.

Comment (yarons) [inline api/budget/user_budget/requirements.txt:1]: Maybe we'll postpone the version specification for after the dev phase?

**Review by yarons — COMMENTED** [review]
You're awesome :)

File: api/budget/common/database/budget_daily_repository.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+from sqlalchemy import and_
+from sqlalchemy.sql import text
+from collections import namedtuple
+
+class BudgetDailyRepository(RepositoryBase):
+    def __init__(self, connection) -> None:
+        super().__init__(connection, BudgetDaily, BudgetDaily.record_id)
+
+    def get_user_budget_daily(self, user_id: str, from_timestamp: datetime, to_timestamp, group_by: str) -> Iterable[dict]:
+        query = text("SELECT date_trunc(:group_by, event_date) AS event_date, avg(amount) as amount FROM budget_daily " +
```
[the remark this answers is not in the document]
Comment (yarons) [reply api/budget/common/database/budget_daily_repository.py:14]: Answered in private, thank you :)

File: local/seed.sql
[the remark this answers is not in the document]
Comment (inbarbarkai) [reply local/seed.sql:1]: I don't want it to be in Liquibase.
It is a seed for local testing.
Shouldn't be used in production.

**Pull request merged**